typescript/ts-converter/src/AstConverter.ts

```diff
@@ -746,7 +742,10 @@ export class AstConverter {
     }
 
     if (Array.isArray(symbol.declarations)) {
-      return symbol.declarations[0];
+      return symbol.declarations.filter(decl => !ts.isModuleDeclaration(decl) &&
```

Comment:
let's try to just find the first one that matches our criteria - the order is guaranteed
 by TypeScript (and if it won't, we'll catch it in tests)